modules/web/src/app/dynamic/enterprise/quotas/component.ts

```diff
@@ -116,6 +123,55 @@ export class QuotasComponent implements OnInit {
     });
   }
 
+  getprogressBarData(projectId: string, resourceType: string): progressBarData {
+    const quota = this.quotas.find(quota => quota.subjectName === projectId);
+    const progressBar: progressBarData = {
+      color: '',
+      value: 0,
+      useageQuota: '',
+    };
+    if (quota) {
+      const percentage = 100;
+      switch (resourceType) {
+        case 'cpu':
+          progressBar.value =
+            ((quota.status.globalUsage.cpu ? quota.status.globalUsage.cpu : 0) / quota.quota.cpu) * percentage;
+          progressBar.color = this._getProgressBarAccent(progressBar.value);
+          progressBar.useageQuota = `${quota.status.globalUsage.cpu ? quota.status.globalUsage.cpu : 0}/${
+            quota.quota.cpu
+          }`;
+          return progressBar;
+        case 'memory':
+          progressBar.value =
+            ((quota.status.globalUsage.memory ? quota.status.globalUsage.memory : 0) / quota.quota.memory) * percentage;
+          progressBar.color = this._getProgressBarAccent(progressBar.value);
+          progressBar.useageQuota = `${quota.status.globalUsage.memory ? quota.status.globalUsage.memory : 0}/${
+            quota.quota.memory
+          }`;
+          return progressBar;
+        case 'storage':
+          progressBar.value =
+            ((quota.status.globalUsage.storage ? quota.status.globalUsage.storage : 0) / quota.quota.storage) *
+            percentage;
+          progressBar.color = this._getProgressBarAccent(progressBar.value);
+          progressBar.useageQuota = `${quota.status.globalUsage.storage ? quota.status.globalUsage.storage : 0}/${
+            quota.quota.storage
+          }`;
+
+          return progressBar;
+      }
+    }
+    return progressBar;
+  }
+
+  private _getProgressBarAccent(percentage: number): ThemePalette {
```

Comment:
we same method in quota-widget component
`getProgressBarAccent` can shared dir `modules/web/src/app/dynamic/enterprise/quotas` that is only available and shared for EE quota module only, thoughts?